src/server/routes/api.js

```diff
@@ -63,4 +63,16 @@ router.post('/buckets', upload.array('screenshots[]', 50), errorChecking(
   },
 ));
 
+router.get('/buckets', errorChecking(
```

Comment:
We could have exposed a graphql query. We might need to do the same for the front-end.